ush/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 list(APPEND ush_files
+  build.sh
```

Comment:
this file should not need be installed.

Reply:
After correcting it in the wrong branch first, I did it right the second time around. thanks!